Sources/Rendering/WebGPU/HardwareSelector/index.js

```diff
@@ -0,0 +1,441 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkHardwareSelector from 'vtk.js/Sources/Rendering/Core/HardwareSelector';
+import vtkWebGPUBuffer from 'vtk.js/Sources/Rendering/WebGPU/Buffer';
+import vtkWebGPUHardwareSelectionPass from 'vtk.js/Sources/Rendering/WebGPU/HardwareSelectionPass';
+import vtkSelectionNode from 'vtk.js/Sources/Common/DataModel/SelectionNode';
+import vtkDataSet from 'vtk.js/Sources/Common/DataModel/DataSet';
+
+const { SelectionContent, SelectionField } = vtkSelectionNode;
+const { FieldAssociations } = vtkDataSet;
+const { vtkErrorMacro } = macro;
+
+function getInfoHash(info) {
```

Comment:
looks familiar? Are some of those functions common between implementations?

Reply:
Some commonality but something like a raytracer would probably not have them. I'm trying to keep the Rendering\Core part to be just the public API. So any common code would have to land somewhere else.

Reply:
Sure it could be in `Rendering/Core/HardwareSelector/helpers.js` if this is more than just that function. I didn't double check but if the `convert` one could be shared and maybe others? 
If not, then don't worry about it.